catalyst/dl/__init__.py

```diff
@@ -9,4 +9,4 @@
 from .experiment import BaseExperiment, SupervisedExperiment, ConfigExperiment
 from .meters import AverageValueMeter, ClassErrorMeter, ConfusionMeter, \
     MSEMeter, MovingAverageValueMeter, AUCMeter, APMeter, mAPMeter
-from .runner import SupervisedRunner
+from .runner import *
```

Comment:
we have to decide our code-style about `*`.

previously, we make an agreement to not use asterisks

Reply:
It would be nice if you could propose workaround. For now - it looks like bad, but working solution.

Reply:
Personally, I prefer to write `__all__` in the script and then in init actually use `from .something import *`

Reply:
But i wonder why you demanded not to write `*` 🤔 in my previous PRs